packages/wallets/src/components/WalletListCardActions/WalletListCardActions.tsx

```diff
@@ -51,22 +63,23 @@ const WalletListCardActions = () => {
         return (
             <div className='wallets-mobile-actions__container'>
                 <div className='wallets-mobile-actions'>
-                    {getWalletHeaderButtons(isDemo).map(button => (
-                        <div className='wallets-mobile-actions-content' key={button.name}>
-                            <IconButton
-                                aria-label={button.name}
-                                className='wallets-mobile-actions-content-icon'
-                                color='transparent'
-                                icon={button.icon}
-                                isRound
-                                onClick={() => {
-                                    history.push(`/wallets/cashier/${button.name}`);
-                                }}
-                                size='lg'
-                            />
-                            <WalletText size='sm'>{button.text}</WalletText>
-                        </div>
-                    ))}
+                    {getWalletHeaderButtons(isDemo).map(button => {
+                        return (
+                            <div className='wallets-mobile-actions-content' key={button.name}>
+                                <IconButton
+                                    aria-label={button.name}
+                                    className={button.className}
+                                    color={button.color}
+                                    icon={button.icon}
+                                    onClick={() => {
+                                        history.push(`/wallets/cashier/${button.name}`);
+                                    }}
+                                    size='lg'
+                                />
+                                <WalletText size='sm'>{button.text}</WalletText>
+                            </div>
+                        );
+                    })}
```

Comment:
Can we remove the additional return to make it more clean
```suggestion
                    {getWalletHeaderButtons(isDemo).map(button => (
                            <div className='wallets-mobile-actions-content' key={button.name}>
                                <IconButton
                                    aria-label={button.name}
                                    className={button.className}
                                    color={button.color}
                                    icon={button.icon}
                                    onClick={() => {
                                        history.push(`/wallets/cashier/${button.name}`);
                                    }}
                                    size='lg'
                                />
                                <WalletText size='sm'>{button.text}</WalletText>
                            </div>
                        ))}
```